src/lib/components/Tree.vue

```diff
@@ -154,6 +156,8 @@ export default {
         parent.indeterminate = !every && every !== some
```

Comment:
Would it still work as expected if we just defined it as below and no longer use parent.indeterminate ?

If it works, I think it would be better to break our dependency on the DOM.

```js
isIndeterminate = !every && every !== some
```

Reply:
Also this condition seems to be wrong. If `every` is true, it will never pass to the right, so we can't expect `every` to be true on the right.

It may be more accurate as below.

```js
isIndeterminate = !every && some;
```